src/main/java/com/transferwise/openbanking/client/json/VrpConsentResponseStatusEnumDeserializer.java

```diff
@@ -0,0 +1,32 @@
+package com.transferwise.openbanking.client.json;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+import com.transferwise.openbanking.client.api.payment.v3.model.vrp.OBDomesticVRPConsentResponseData.StatusEnum;
+import java.io.IOException;
+
+public class VrpConsentResponseStatusEnumDeserializer
+    extends StdDeserializer<StatusEnum> {
+
+    protected VrpConsentResponseStatusEnumDeserializer() {
+        this(null);
+    }
+
+    protected VrpConsentResponseStatusEnumDeserializer(Class<?> vc) {
+        super(vc);
+    }
+
+    @Override
+    public StatusEnum deserialize(JsonParser jp, DeserializationContext ctxt)
+        throws IOException {
```

Comment:
Why this is on a new line?

Reply:
Fixed in [7a5751f](https://github.com/transferwise/openbanking-client/pull/180/commits/7a5751fbfdc46c88d1889807a5865b4b691270a4).